f/radio/f_radioChannels.sqf

```diff
@@ -0,0 +1,128 @@
+// F3 - Folk ARPS Long-Range Radio Module
+// Credits and documentation: https://github.com/folkarps/F3/wiki
+
+/* ========================
+This module is activated in init.sqf.
+Example:
+[true] execVM "f\radio\f_radioChannels.sqf";
+
+Arguments:
+0. Long-range split mode (boolean). If true, each radio backpack type has a separate channel. If false, all backpacks use one channel. Default false.
+
+This module requires some missionmaker configuration. If you want to enable vehicle radio access, you need to specify vehicle classes or variable names below.
+=========================== */
+
+params [["_splitMode",false]];
```

Comment:
It was more important in the first iteration of the system (before the vehicle radio refactor) when the channels were generated differently. So in some ways it's a holdover. However, I think having the ability to use a simple toggle rather than futzing with arrays can be useful. The radio system does work in the default unconfigured state, using only backpacks and no vehicles, so if that's all that's required for a mission, setting the split mode is all they need to do to be up and running. It also provides missionmakers the ability to quickly reuse a set of eligible items in a different mode, without needing to change the arrays, as well as a simple modular hook to connect to e.g. a mission parameter.